### ISSUE TITLE
(PROD061)  - (REP) -  Missing 'Active Case' checkbox

### ISSUE THREAD

**Issue by anthonyperk** [CurrentlyNotSet/SERB#437]
The active case checkbox is missing from SMDS 3.0
In the old system (2.0), I had at the bottom left of the screen on the Information Tab, and check box that stated “Active”. When I closed cases, I unchecked the box. I do not have it now.

![image](https://cloud.githubusercontent.com/assets/24595073/24724622/c5fff866-1a19-11e7-8399-0f05af1d73a6.png)

**Comment by altrdgenetics** [comment]
Email to Justin:

> Justin,
> 
> This ticket confuses me. During the meeting that XLN had with you and Chris we discussed this exact field. We were told back in the beginning while demoing to the REP section that it was a duplicate of the "Status1" drop-down which contained the options "Open/Closed". The reason that we brought this up during the meeting when discussing existing open tickets was due to its usage a single report that Anthony was investigating how to write the statement. The solution provided below is from Anthony per our discussion during that meeting:
> 
> "For the reports, the Inactive Election Report in 2.0 used this checkbox in part of its WHERE statement (repdata."CaseActive" = '0').  The Active Cases Status Report did NOT use this checkbox at all.  In the 3.0 reports, the Inactive Election Report uses WHERE the final board date is not blank and the tally date/ballotscountdate for the date range (the Active Election Report uses
>     WHERE
>     REPCase.pollingStartDate IS NOT NULL
>     AND REPCase.pollingEndDate IS NOT NULL
>     AND REPCase.active = '1'
>     AND REPCase.status1 = 'Open'
>     AND REPCase.finalBoardDate IS NULL)
> "
> 
> 
> After explaining the situation to you, you agreed that the field is a duplicate and was indeed not needed in the system. The ticket itself says it is used when closing out cases, isn't that a duplicate function of the "status 1" drop-down?
> 
> note: if this is to be added back into the system a database change will be required.
> 
> - Andrew

Response:

> Andrew,
>  
> Thank you for the quick response. I will double check to see if it is needed.
>  
> Thank you,
>  
> Justin Brown

**Comment by parkjohnston** [comment]
Time to start billing for all the waffling

**Comment by altrdgenetics** [comment]
Sent Email To Justin Reminding him we have a outstanding question on this ticket.

**Comment by altrdgenetics** [comment]
Email From Justin:

> PROD061 - Waiting on information regarding if checkbox is needed
> (Still investigating)

**Comment by altrdgenetics** [comment]
When discussing this with Justin today he said that he made several attempts to get in contact with Angela about it. No response was given and she has chosen silent protest by being out of the office 3-4 days a week since the system has gone live.
